### PR TITLE
upd #170936

### PR DESCRIPTION
**Pull request by Yuki2718**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [x] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/pull/170936

### Add your comment and screenshots

1. Your comment

Let's expand the coverage. IDK if the `www.` issue on iOS was fixed, if it is, the rule can be changed to `www.`. I didn't add
```
youtube.com,~accounts.youtube.com,~m.youtube.com,~payments.youtube.com,~tv.youtube.com,~youtubekids.com,~youtube-nocookie.com#%#//scriptlet('trusted-replace-node-text', 'script', 'ytInitialPlayerResponse', '/"auxiliaryUi":\{"messageRenderers":\{"bkaEnforcementMessageViewModel.*?e\}\}\}\,/', '')
youtube.com,~accounts.youtube.com,~m.youtube.com,~payments.youtube.com,~tv.youtube.com,~youtubekids.com,~youtube-nocookie.com#%#//scriptlet('trusted-replace-node-text', 'script', 'ytInitialPlayerResponse', '("trackingParam":"kx_fmPxhoPZR)[-_0-9A-Za-z]{150}[-_0-9A-Za-z]+?([-_0-9A-Za-z]{43}BwOcCE59TDtslLKPQ-SS"\})/', '$1$2')
```
as they never worked, meaning user may see the warning after the first load. `set-constant` alternatives OTOH cause error for some reason.

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -180,6 +180,11 @@ youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', '
 /youtubei/v*/player?key=$replace=/\"playerAds.*?\}\}\]\,//,domain=youtubekids.com|youtube-nocookie.com|youtube.com
 !#endif
 !#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android && !adguard_ext_firefox)
+! https://github.com/AdguardTeam/AdguardFilters/pull/170936
```
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:183]: Cannot check where they should be applied, but it seems nothing is broken (checked on desktop, android and ios)

Comment (Yuki2718) [reply BaseFilter/sections/specific.txt:183]: This PR is just a scriptlet version of https://github.com/AdguardTeam/AdguardFilters/pull/170936 to cover Chrome and Safari extension.

Comment (Alex-302) [reply BaseFilter/sections/specific.txt:183]: @AdamWr @piquark6046 Please also take a look.

Comment (AdamWr) [reply BaseFilter/sections/specific.txt:183]: After quick test I don't see any breakage on my end with these rules.

By the way, if I'm not wrong, rule like `~tv.youtube.com,youtube.com#%#` doesn't work in Safari on both websites.

Comment (Alex-302) [reply BaseFilter/sections/specific.txt:183]: yep

Comment (Yuki2718) [reply BaseFilter/sections/specific.txt:183]: Can I change to `www.youtube.com` then? https://github.com/AdguardTeam/AdguardForiOS/issues/1897 is closed.

Comment (Alex-302) [reply BaseFilter/sections/specific.txt:183]: Will check on iOS.
Is/was something broken on accounts.youtube.com, m.youtube.com, payments.youtube.com and tv.youtube.com?

Comment (Alex-302) [reply BaseFilter/sections/specific.txt:183]: m.youtube.com, music.youtube.com works fine

Comment (Yuki2718) [reply BaseFilter/sections/specific.txt:183]: > Is/was something broken 

No, but as the anti-adb is shown only on www it'll be best to inject only on www.

**Review by Alex-302 — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

**Pull request merged**